test/splitting.jl

```diff
@@ -0,0 +1,51 @@
+using LazySets, Test, LinearAlgebra
+using NeuralVerification
+import NeuralVerification: ReLU, Id
+
+macro no_error(ex)
+    quote
+        try $(esc(ex))
+            true
+        catch
+            false
+        end
+    end
+end
+
+@testset "Splitting Test" begin
+
+    net_file = "$(@__DIR__)/../examples/networks/R2_R2.nnet"
+    # This is a 3 layer simple network mapping from R2 to R2.
+
+    # y1 =  x1 + x2 - 1
+    # y2 = -x1 + x2 - 1
+
+    # z1 = ReLU(y1)
+    # z2 = ReLU(y2)
+
+    # o1 = 1*z1 + 0*z2 + 0
+    # o2 = 0*z1 + 1*z2 + 0
+
+    net = read_nnet(net_file, last_layer_activation = Id())
+    A = [1. 0.; 0. 1.; -1. 0.; 0. -1.;]
+    b = [1.1, 1.1, 0.1, 0.1]
+    X = Hyperrectangle(low=[-1,-1], high=[1,1])
+    # The true output set is a union of two segments : (0,0)->(0,1) U (0,0)->(1,0)
+    Y = HPolytope(A, b) # Output constraints. Hyperrectangle: -0.1, -0.1, 1.1, 1.1
+    problem = Problem(net, X, Y)
+
+    @testset "Neurify" begin
+        solver = Neurify()
+        @test @no_error result = solve(solver, problem)
```

Comment:
Why is it testing no_error here? If the issue is that unknown can come up, we should allow for it explicitly like in the other tests.

Reply:
I thought `no_error` is a must be to convert a running status to a `true` or `false` bool. 
Looks like I'm wrong. I will remove it.

Reply:
It simply tests that a line didn't produce an error. We used it once or twice on tests that we didn't have a confirmed answer for yet but wanted to see that the code at least ran. We're phasing it out where possible.